src/Baseline/Formatter/BaselineErrorFormattable.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace TwigStan\Baseline\Formatter;
+
+use TwigStan\Application\TwigStanAnalysisResult;
+
+interface BaselineErrorFormattable
```

Comment:
I would call it Formatter:
```suggestion
interface BaselineErrorFormatter
```

But maybe this is better:
```suggestion
interface BaselineFormatter
```